avoid that full path to bin/python is used in Python scripts
author: Kenneth Hoste (HPC-UGent)
--- gobject-introspection-1.46.0.orig/tools/g-ir-tool-template.in	2015-10-27 12:42:43.844025247 +0100
+++ gobject-introspection-1.46.0/tools/g-ir-tool-template.in	2015-10-27 12:41:09.431855691 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env @PYTHON@
+#!/usr/bin/env python
 # -*- Mode: Python -*-
 # GObject-Introspection - a framework for introspecting GObject libraries
 # Copyright (C) 2008  Johan Dahlin
